Intersect.Client.Framework/Database/GameDatabase.cs

```diff
@@ -158,6 +161,7 @@ public virtual void SavePreferences()
         SavePreference(nameof(FullScreen), FullScreen);
         SavePreference(nameof(ShowFPSCounter), ShowFPSCounter);
         SavePreference(nameof(EnableLighting), EnableLighting);
+        SavePreference(nameof(EnableMouseScrollZoom), EnableMouseScrollZoom);
```

Comment:
I think it should be named `EnableScrollingWorldZoom` to match the original feature's preference name